src/lib.rs

```diff
@@ -1912,6 +1912,19 @@ impl Build {
                         cmd.push_cc_arg("-fno-plt".into());
                     }
                 }
+                if target.contains("wasm32-wasi") {
+                    // WASI does not support exceptions yet.
+                    cmd.push_cc_arg("-fno-exceptions".into());
+                    let wasi_sdk = std::env::var("WASI_SDK").expect("Could not find WASI_SDK. Download it from github & setup environment variable WASI_SDK targetting the folder.");
```

Comment:
Please use `var_os` since the path can contain non utf-8 characters:

```suggestion
                    let wasi_sdk = std::env::var_os("WASI_SDK").expect("Could not find WASI_SDK. Download it from github & setup environment variable WASI_SDK targetting the folder.");
```